### ISSUE TITLE
Cannot publish site with new images.

### ISSUE THREAD

**Issue by mikaelsand** [Azure/api-management-developer-portal#32]
**Describe the bug**
Image files become XML-files with errors after publish.

**To Reproduce**
I follow the how to guide to create a new design. I upload new pictures, I leave the site running and run the npm run publish. This seems to pull image files from blob storage and store them in the website/content folder. The files become 1 kb large jpgs.
I opened the jpgs in VS and saw that they are xml files, containing errors.

`<?xml version="1.0" encoding="utf-8"?><Error><Code>ResourceNotFound</Code><Message>The specified resource does not exist.
RequestId:c87ba921-701e-0036-1776-d259a6000000
Time:2019-03-04T10:39:08.2825033Z</Message></Error>`

**Expected behavior**
Files should be jpegs.

**Screenshots**
I also have the run log from chrome showing som additional information from when I added a new image to the site:
![image](https://user-images.githubusercontent.com/20858407/53734529-85a06f80-3e84-11e9-93e0-00f21656e82b.png)

**Additional context**
Basically it seems it cannot get the file from blob storage when the publish script runs.
